docs/dev/run_the_etl.rst

```diff
@@ -222,19 +222,7 @@ the job. The bottom part of the window contains dagster logs. You can
 view logs from the ``pudl`` package in the CLI window the ``dagster-webserver`` process
 is running in.
 
-If you need to set op configurations, such as the ``clobber`` setting, you can
-add them in the Launchpad tab of the job like so::
```

Comment:
Is there another example that we should put in here instead?

Reply:
I feel like the only times I ever used this were for `clobber` - nothing comes to mind.